templates/index.zh_cn.html

```diff
@@ -0,0 +1,275 @@
+                        <li>无合同</li>
+                        <li>无许可费</li>
+                        <li>不削减销售额</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="feature-image">
+                <img src="/assets/opensource.svg" class="feature-img" alt="开源图标" />
+            </div>
+        </div>
+    </div>
+    <div class="media-content features-whats-next">
+        准备好开始构建 Bevy 应用程序了吗？从 <a href="learn/book/introduction/">The Bevy Book</a> 开始快速入门！
+    </div>
+    <img src="/assets/bevy_logo_dark.svg" class="bevy-logo-header" alt="Bevy 徽标" />
+    <div class="features-sponsors">
+        <h3 class="media-content">在此，我们衷心感谢那些对 Bevy 慷慨解囊的<a
+                href="https://github.com/sponsors/cart/">赞助商们</a>，是他们使得我们的工作得以持续开展。
+                请注意，所有接受赞助的 Bevy 领导者都签署了此<a
+                href="/sponsorship-pledge">赞助承诺书</a>，以确保 Bevy 始终是一个社区优先的项目。 
+        </h3>
+        <!-- <div class="sponsors sponsors--patron">
+            <h2 class="sponsors__title">Patron Sponsors</h2>
+            <h3 class="sponsors__amount">Over $100,000 / year</h3>
+            <div class="sponsors__content">
+            </div>
+        </div> -->
+        <div class="sponsors sponsors--platinum">
+            <h2 class="sponsors__title">铂金赞助商</h2>
+            <h3 class="sponsors__amount">$1,000 / 月</h3>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://www.foresightmining.com">
+                    <img class="sponsors__logo" style="height: 140px" src="/assets/Foresight_Spatial_Labs.svg"
+                        alt="Foresight Spatial Labs 徽标" />
+                </a>
+                <a class="sponsors__link" href="https://www.embark-studios.com">
+                    <img class="sponsors__logo" src="/assets/embark.png" alt="Embark Studios 徽标" />
+                </a>
+                <a class="sponsors__link" href="https://meetkai.com/careers">
+                    <img class="sponsors__logo" style="max-height: 120px" src="/assets/meetkai.png" alt="MeetKai 徽标" />
+                </a>
+                <a class="sponsors__link" href="https://www.encultured.ai/">
+                    <img class="sponsors__logo" style="max-height: 60px" src="/assets/Encultured.png"
+                        alt="Encultured AI 徽标" />
+                </a>
+            </div>
+        </div>
+        <div class="sponsors sponsors--gold">
+            <h2 class="sponsors__title">黄金赞助商</h2>
+            <h3 class="sponsors__amount">$300 / 月</h3>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://playroids.com">
+                    <img class="sponsors__logo" src="/assets/roids_logo.png" alt="Roids 徽标" style="max-height: 130px" />
+                </a>
+            </div>
+        </div>
+        <div class="sponsors sponsors--silver">
+            <h2 class="sponsors__title">白银赞助商</h2>
```

Comment:
I'm not sure its viable for us to keep all translated sponsor lists in sync. It might be better to leave this out until we can automate it or something.